vmdb/app/models/mixins/naming_sequence_mixin.rb

```diff
@@ -0,0 +1,17 @@
+module NamingSequenceMixin
+  extend ActiveSupport::Concern
+
+  included do
+    # these are possible naming sequences because there is no constraint to source column
+    has_many :naming_sequences, :as => :resource, :dependent => :destroy, :class_name => "CustomAttribute"
```

Comment:
I don't understand the comment.

Also, is there one or many of these naming_sequences? (the `next_naming_sequence` block seems to suggest this is a `has_one`)